.buildkite/bindists-pipeline

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+BAZEL_DIR="$(.buildkite/fetch-bazel-bindist)"
+trap "rm -rf '$BAZEL_DIR'" EXIT
+export PATH="$BAZEL_DIR:$PATH"
+echo "common:ci --build_tag_filters -requires_lz4,-requires_proto,-requires_zlib,-requires_doctest,-requires_c2hs,-requires_threaded_rts,-dont_test_with_bindist" > .bazelrc.local
+# XXX: See .bazelrc [backward compatible options] for the the rational behind this flag
+echo "build --incompatible_use_python_toolchains=false" >> .bazelrc.local
+echo "build:ci --experimental_strict_action_env" >> .bazelrc.local
```

Comment:
There ought to be a note here that

> Enables strict action env in the bindists pipeline.

is a temporary workaround. We should no longer do anything that would require `strict_action_env`, now that we have the means to achieve that, via the rules_sh toolchain.

Reply:
I added a comment to that effect.